reference/pkg/nodejs/@pulumi/pulumi/cmd/index.md

```diff
@@ -36,6 +36,6 @@ title: Module cmd
 </h2>
 <div class="pdoc-module-contents" markdown="1">
 
-<pre class="highlight"><span class='kd'></span>run(argv: ParsedArgs): <span class='kd'><a href='https://www.typescriptlang.org/docs/handbook/basic-types.html#void'>void</a></span></pre>
+<pre class="highlight"><span class='kd'></span>run(argv: ParsedArgs, programStarted: () => <span class='kd'><a href='https://www.typescriptlang.org/docs/handbook/basic-types.html#void'>void</a></span>): <span class='kd'><a href='https://www.typescriptlang.org/docs/handbook/basic-types.html#void'>void</a></span></pre>
```

Comment:
We should probably be marking this and many other things in this sub-package as internal...